customer/interfaces/graphql/schema.graphql

```diff
@@ -4,7 +4,9 @@ type Commerce_Customer_Status_Result {
 }
 
 type Commerce_Customer_Result {
-    id: String!
+    id: String! @deprecated(reason: "Use `userID` field.")
+    userID: String!
```

Comment:
What is the difference between userID and id here? Why is it deprecated? "Use another field" is not a reason

Reply:
removed.